### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by peci1** [ros-perception/laser_filters#131]
Could I ask for a new Melodic/Noetic release? There seems to have been quite some development since the last one (more than a year now).

**Comment by jonbinney** [comment]
Yes - I've got a couple PRs I want to merge first, then I'll release. I've created a noetic-devel branch so that I could merge the header path changes in https://github.com/ros-perception/laser_filters/pull/128 .

@peci1 are you actively using both melodic and noetic? I'm trying to decide how aggressively I should backport commits from noetic-devel to the indigo-devel branch so that they get released into melodic. (melodic gets released from the inidigo-devel branch)

**Comment by peci1** [comment]
Thanks. We're actively using Melodic and will be slowly switching to Noetic, so we're interested in having the changes in both. Melodic still has 2 years of lifetime, so I think it makes sense to backport things.

**Comment by jonbinney** [comment]
Melodic release here: https://github.com/ros/rosdistro/pull/31119

I've updated the source and doc branches to match the branch that is actually released into melodic: kinetic-devel.

After that PR gets merged I'll do the noetic release from the new noetic-devel branch.

**Comment by jonbinney** [comment]
Noetic release here: https://github.com/ros/rosdistro/pull/31149. Noetic version numbers are now `1.9.x` to distinguish them from kinetic versions which are `1.8.x`
